src/utils/index.js

```diff
@@ -105,3 +108,23 @@ export const calendarMonthsToRender = (
   }
   return R.compose(R.take(visibleMonths), R.drop(currentMonth))(months);
 };
+
+type CustomClasses = { [className: string]: Date[] };
+
+export const filterCustomClasses = (
```

Comment:
add pls some comment what this all code do in general

Reply:
done 👊